sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/models/event_data.hpp

```diff
@@ -79,6 +74,7 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Models {
      */
     EventData& operator=(EventData&&) = default;
   };
+  std::ostream& operator<<(std::ostream&, EventData const&);
```

Comment:
Doc comment missing.

Reply:
doxygen doesn't seem to care, and do we *really* want to add documentation of the ostream inserter?